tac/agents/controller/agent.py

```diff
@@ -96,10 +114,16 @@ def act(self) -> None:
             logger.debug("[{}]: waiting for starting the competition: start_time={}, current_time={}, timedelta ={}s"
                          .format(self.name, str(self.game_handler.tac_parameters.start_time), str(now), seconds_to_wait))
             self.game_handler.competition_start = now + datetime.timedelta(seconds=seconds_to_wait + self.game_handler.tac_parameters.registration_timedelta.seconds)
+
             time.sleep(seconds_to_wait)
             logger.debug("[{}]: Register competition with parameters: {}"
                          .format(self.name, pprint.pformat(self.game_handler.tac_parameters.__dict__)))
             self.oef_handler.register_tac()
+
+            self.set_shared_status("Registration open")
```

Comment:
@dishmop I suggest creating something like 
`
class GameStatus(Enum):
     REGISTRATION_OPEN = 'registration_open'
    ....
`
somewhere. Then we have one place where we can see the different status options.

Reply:
Like you do with `SandboxState` below :)